doc/src/manual/conversion-and-promotion.md

```diff
@@ -168,13 +168,12 @@ Such a definition might look like this:
 convert(::Type{MyType}, x) = MyType(x)
 ```
 
-The type of the first argument of this method is a [singleton type](@ref man-singleton-types),
+The type of the first argument of this method is a [`Type{T}`](@ref man-typet-type),
 `Type{MyType}`, the only instance of which is `MyType`. Thus, this method is only invoked
```

Comment:
Why not put directly `Type{MyType}` within the `[]` brackets, where `Type{T}` is currently?